docs/release_notes/v0.9.1-alpha.md

```diff
@@ -0,0 +1,30 @@
+The v0.9.1 release adds `flightsql` data connector and `sqlite` data store. Improved the usability by adding more `spice` CLI commands.
+
+## Highlights in v0.9.1-alpha
+
+**FlightSQL data connector**: [Arrow Flight SQL](https://arrow.apache.org/docs/format/FlightSql.html) can now be used as a connector for federated SQL query.
+
+**SQlite data backend**: [SQLite](https://www.sqlite.org/) can now be used as a data store for acceleration.
+
+## Contributors
+
+- @phillipleblanc
+- @mitchdevenport
+- @Jeadie
+- @ewgenius
+- @sgrebnov
+- @y-f-u
+- @lukekim
+
+## New in this release
+
+- **Adds** FlightSQL data connector (`flightsql`).
+- **Adds** SQLite data store (`sqlite`).
+- **Adds** support for date, varchar, bpchar, and primitive list types for the PostgreSQL data connector and data store.
+- **Adds** `spice pods`, `spice status`, `spice datasets`, and `spice models` CLI commands.
+- **Adds** `GET /v1/spicepods` API for listing loaded Spicepods.
+- **Adds** `spiced` Docker CI build and release.
+- **Adds** E2E tests for release installation and local acceleration.
```

Comment:
Please add
- **Adds** E2E tests and instructions to run basic [TPC-H benchmark](https://www.tpc.org/tpch/) tests